docs/BEMAN_STANDARD.md

```diff
@@ -171,6 +187,20 @@ add_executable(beman.smart_pointer.tests.roundtrip)
 #...
 ```
 
+**[CMAKE.PASSIVE_TARGETS]** REQUIREMENT: External targets must not modify
+compilation flags of dependents.
+
+Therefore, `target_compile_features` (e.g., `cxx_std_20`) must not be used
+because it modifies the compilation environment of dependent targets. Compiler
+support for required features should be determined at CMake configuration time
+using `check_cxx_source_compiles`.
+
+Furthermore, `target_compile_definitions` with `PUBLIC` or `INTERFACE`
+visibility must not be used, as these definitions are also propagated to
+dependent targets. Preprocessor definitions intended for external use should be
+generated into a `config.hpp` file at CMake configuration time. This
+`config.hpp` should then be included by public headers.
```

Comment:
Is there something we can reference to?

Reply:
👌 cool, just checking. We just need to add an example once someone implements this.